protocol/communities/manager.go

```diff
@@ -3240,7 +3271,11 @@ func (m *Manager) PendingRequestsToJoin() ([]*RequestToJoin, error) {
 }
 
 func (m *Manager) PendingRequestsToJoinForUser(pk *ecdsa.PublicKey) ([]*RequestToJoin, error) {
-	return m.persistence.PendingRequestsToJoinForUser(common.PubkeyToHex(pk))
+	return m.persistence.RequestsToJoinForUserByState(common.PubkeyToHex(pk), RequestToJoinStatePending)
+}
+
+func (m *Manager) AwaitingRequestsToJoinForUser(pk *ecdsa.PublicKey) ([]*RequestToJoin, error) {
```

Comment:
```suggestion
func (m *Manager) AwaitingAddressesRequestsToJoinForUser(pk *ecdsa.PublicKey) ([]*RequestToJoin, error) {
```

Otherwise `AwaitingRequestsToJoinForUser` vs `PendingRequestsToJoinForUser` is very confusing.